README.md

```diff
@@ -3,132 +3,339 @@
-- The orientation estimation Roll, Pitch and Yaw.
-- The accelerometer measuremtents x, y, z components.
-- The gyroscope measuremtents x, y, z components.
-- The magnetometer measuremtents x, y, z components.
+The Control Console GUI displays a series of buttons which trigger the same commands usually sent through the RPC interface running on a terminal.
+These commands are implemented with the most common options, which can be set through editable text input forms.
 
 ## [Dependencies](#top)
 
-Server dependencies:
-- [YARP](https://github.com/robotology/yarp): Middleware for handling the communication with the robot.
-- [node.js](https://nodejs.org/en/): Asynchronous event-driven JavaScript runtime, designed to build scalable network applications.
+### Server dependencies
+- [NVM](https://github.com/nvm-sh/nvm): Node Version Manager.
+- [Node.js](https://nodejs.org/en/): Asynchronous event-driven JavaScript runtime, designed to build scalable network applications.
 - [npm](https://www.npmjs.com/): Node package manager.
-- [YarpJS](https://github.com/robotology/yarp.js): Yarp Javascript bindings
 - [Open MCT](https://github.com/nasa/openmct): Open source visualization framework by NASA.
+- [YarpJS](https://github.com/robotology/yarp.js): Yarp Javascript bindings
+- [YARP](https://github.com/robotology/yarp): Middleware for handling the communication with the robot.
+- [CMake](http://www.cmake.org/download/)
+- A proper C/C++ compiler toolchain for the given platform
+    - Windows:
+        - [Visual C++ Build Tools](https://visualstudio.microsoft.com/visual-cpp-build-tools/) or a recent version of Visual C++ (e.g. [the free Community](https://www.visualstudio.com/products/visual-studio-community-vs))
+    - Unix/Posix:
+        - Clang or GCC
+
+#### Notes
+- Additional dependencies [Open MCT](https://github.com/nasa/openmct) (Open source visualization framework by NASA) and [YarpJS](https://github.com/robotology/yarp.js) (Yarp Javascript bindings) are automatially installed by the Node.js `npm` package manager when installing the repository main modules (steps 7 and 8).
+- [YARP](https://github.com/robotology/yarp), [CMake](http://www.cmake.org/download/) and C/C++ compiler toolchain are transitory dependencies which shouldn't be mentioned a priori here, but since they are not handled by the [YarpJS](https://github.com/robotology/yarp.js) installation process (via the package manager `npm`), they need to be manually installed.
 
-Client dependencies:
+### Client dependencies
 - Browser: [Google Chrome](https://www.google.com/chrome), [Mozilla Firefox](https://www.mozilla.org/en-US/firefox/products), Apple Safari, etc.
 
+### Simulation Dependencies
+- [Gazebo](https://gazebosim.org): the simulator (tested Gazebo 11).
+- [YARP](https://github.com/robotology/yarp): Middleware for handling the communication with the robot.
```

Comment:
I think this is repeated. ``yarp`` is already a dependency

Reply:
I was considering a standalone standalone setup running the simulation. I had in mind here the use case where you run a Gazebo simulation in some machine, for testing some control algorithm for instance, and you run the telemetry server on your machine to get the simulation data from the network. But I agree this can be confusing for the reader.

I will consider in the dependencies section only:
- the visualization tool server and client mandatory dependencies
- the optional dependencies ("optional" as you mentioned in some later comment in your review) for testing the installed visualization tool itself. In such case, indeed, I consider a simulation setup installed on the same machine, to keep it simple.
I guess that's what you were expecting when reading this section right?

Reply:
...and if we consider the optional dependencies on the same machine, YARP is a duplicate here, as you mentioned.

Reply:
Ah I see, you are right. I would just mention in that line that you need to install it only if you are running the simulation on another machine.

Reply:
As we discussed in the call, I considered the additional dependencies for testing the visualization tool, not for installing a simulation setup in a standalone machine. So I just removed the YARP line.